src/types/Privacy.ts

```diff
@@ -83,4 +83,24 @@ export type PrivacyType = {
      * @note The debug geography is used only when the app is in debug mode.
      */
     setConsentFlowDebugUserGeography(userGeography: ConsentFlowUserGeography): void;
+
+    /**********************************************************************************/
+    /* Google-certified CMP */
+    /**********************************************************************************/
+
+    /**
+     * Shows the CMP flow to an existing user.
+     * Note that the user's current consent will be reset before the CMP alert is shown.
+     *
+     * The function returns when the flow finishes showing.  On success, returns nothing.  On
+     * failure, returns one of the CmpError code.
```

Comment:
@PLP-Consulting - any suggestions? (e.g. we don't use two spaces after period, and maybe "... returns an error code defined in {CmpError}.`)

Reply:
- Are we trying to follow the TSDoc spec for our code comments? If so, the last paragraph should probably be in a `@returns` section.
- and also, if we're going by TSDoc, `{@link CmpError}` is probably better than just `CmpError` in the last line.
- In the second sentence, I recommend replacing "will be reset" and "is shown" with active voice phrases that reveal the identity of the actors. (See [this guide](https://support.applovin.com/hc/en-us/articles/11623042835213#h_01HCFRW40TMNTSYAA4RY057Z3P) for more of the reasoning behind this.) E.g. "Note that Simon resets the user's consent state to Flibbertigibbet before Garfunkle shows the CMP alert to the user."
- Yeah, two periods after punctuation is unnecessary.
- "returns null" is more precise and so probably better than "returns nothing"
- "one of the {@link CmpError} code**s**" (plural)